### DISCUSSION TITLE
Welcome to PowerPlatformAF Discussions!

### DISCUSSION THREAD

**Discussion by andrewdwelch** [PowerPlatformAF/PowerPlatformAF#94] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Customery** [comment]
How does the Power Platform Adoption Framework's enterprise management maturity model compare with Microsoft's [Power Platform adoption maturity model](https://docs.microsoft.com/en-us/power-platform/guidance/adoption/maturity-model)? There seems to be a fair degree of overlap and I'm wondering if we need two ways of assessing maturity.

**Comment by azimzicar** [comment]
All adoption frameworks and maturity models will have some type of overlap as they all trying to achieve the same goal of providing guidance on how to adopt and measure Power Platform in the organisations.

Best thing to so is know the different frameworks and maturity models and then decide what's best for you.

Regards
Azim Zicar - www.zicarconsultancy.co.uk

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by andrewdwelch** [reply]
Totally agree with @azimzicar. I'll add though that I think the "Enterprise Management Maturity Model" in the Adoption Framework is specifically focused on "enterprise management", which I like to define as "infrastructure + governance of that infrastructure". It is probably more IT-focused, and I've had a lot of success using it to work through challenges with IT and information security teams in scenarios where the org wants to adopt Power Platform as a first-class component of their overall app and data ecosystem. I've written a piece recently on the Power Platform Landing Zone that I need to add into the framework... just have not had the time (see link below)...

https://www.andrewdwelch.com/blog/landing-zone
